bin/win-software-setup.ps1

```diff
@@ -134,4 +133,12 @@ else {
         scoop bucket add github-gh "https://github.com/cli/scoop-gh.git"
         scoop install gh
     }
+
+    # Install Python
+    if (Get-Command -Name "python" -ErrorAction SilentlyContinue) {
```

Comment:
Lewis: On my computer this is going to cause an error, because on Winblows 10 there is a "fake python" installed here: `/c/Users/pat/AppData/Local/Microsoft/WindowsApps/python`. This is actually a stub application for the Winblows App Store:

```
$ python
bash: /c/Users/pat/AppData/Local/Microsoft/WindowsApps/python: Permission denied
```

Reply:
Should I just call Install and not bother with Update then?

Reply:
Yes, that seems fine. I tried running install again after install and it worked fine on my computer.